dev-db/postgrest/postgrest-0.4.4.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit systemd user
+
+DESCRIPTION="serves a fully RESTful API from any existing PostgreSQL database."
+HOMEPAGE="https://postgrest.com"
+SRC_URI="https://github.com/begriffs/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="dev-haskell/stack"
+RDEPEND="!dev-db/postgrest-bin
+	dev-db/postgresql
```

Comment:
The docs state 9.5 is the minimum. You may want to consider inheriting the postgres eclass.